config/initializers/swagger_blocks.rb

```diff
@@ -0,0 +1,34 @@
+module ApiFlashcards
+  include Swagger::Blocks
+    swagger_root do
+      key :swagger, '2.0'
+      info do
+        key :version, '1.0.0'
+        key :title, 'Flashcards API'
+        key :description, 'A sample API for Flashcards App'
+      end
+      tag do
+        key :name, 'Cards'
+      end
+      tag do
+        key :name, 'Review'
+      end
+      key :host, 'localhost:3000'
+      key :basePath, '/api/v1'
+      key :produces, ['application/json']
+    end
+
+    SWAGGERED_CLASSES = [
+      Api::V1::CardsController,
+      Api::V1::ReviewController,
+      Card,
+      ErrorModel,
+      self,
+    ].freeze
+
+    #Prepares and writes json for Swagger-UI(swagger_engine gem)
+    swagger_data = Swagger::Blocks.build_root_json(SWAGGERED_CLASSES)
+    File.open(Rails.public_path.to_s + "/api/v1/docs.json", 'w') do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.